Mathlib/CategoryTheory/ChosenFiniteProducts/Cat.lean

```diff
@@ -0,0 +1,98 @@
+/-
+Copyright (c) 2024 Nicolas Rolland. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Nicolas Rolland
+-/
+import Mathlib.CategoryTheory.ChosenFiniteProducts
+/-!
+# Chosen finite products in `Cat`
+
+This file proves that the cartesian product of a pair of categories agrees with the
+product in `Cat`, and provides the associated `ChosenFiniteProducts` instance.
+-/
+
+universe v u
+
+namespace CategoryTheory
+
+namespace Cat
+
+open Limits
+
+/-- The chosen terminal object in `Cat`. -/
+abbrev chosenTerminal : Cat := Cat.of (ULift (ULiftHom (Discrete Unit)))
+
+example : chosenTerminal := ⟨⟨⟨⟩⟩⟩
```

Comment:
What is this supposed to communicate to the reader?

Reply:
This example should be removed.